From 81d916204f68304246a688111a219ec86b259fec Mon Sep 17 00:00:00 2001
From: Diego Nicola Barbato <dnbarbato@posteo.de>
Date: Wed, 4 Aug 2021 17:18:03 +0200
Subject: [PATCH 46/62] configure: Check for wolfSSL instead of OpenSSL.

Since commit 73e53a2 VDE CryptCab is linked with wolfssl (provided by
wolfSSL) instead of crypo (provided by OpenSSL).  Adjust the
corresponding check accordingly.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2aedfd984..0b1132c25 100644
--- a/configure.ac
+++ b/configure.ac
@@ -21,7 +21,7 @@ AC_DEFINE_UNQUOTED(MODULES_EXT, "$shrext_cmds", [Extension of shared objects])
 
 # Checks for libraries.
 AC_CHECK_LIB([dl], [dlopen])
-AC_CHECK_LIB([crypto], [EVP_EncryptInit],
+AC_CHECK_LIB([wolfssl], [wc_Chacha_Process],
   [add_cryptcab_support=yes],
   [add_cryptcab_support=no ; warn_cryptcab=yes])
 AC_CHECK_LIB([pthread], [pthread_create],
-- 
2.35.0

